static/js/public/snap-details/blog-posts.js

```diff
@@ -83,13 +91,15 @@ function snapDetailsPosts(
     throw new Error("Snap not defined");
   }
 
+  this.limit = blogPosts.holder.dataset.limit;
```

Comment:
Hmm... I wonder why don't we do more of that instead passing lots of options via JSON to function. But I guess this works only for a small settings like a number or short string.

Anyway, shouldn't there be some default value if it's not set in DOM?

Reply:
I used `this.limit` rather than `blogPosts.limit`. I thought the function was in the class. The class does have a default of 3 though :)